c/httpserver.c

```diff
@@ -1777,9 +1749,7 @@ static void writeXmlByteCallback(xmlPrinter *p, char c){
 
 xmlPrinter *respondWithXmlPrinter(HttpResponse *response){
   if (response->responseTypeChosen){
-#ifdef DEBUG
-    printf("*** WARNING *** response type already chosen\n");
-#endif
+    zowelog(NULL, LOG_COMP_HTTPSERVER, ZOWE_LOG_DEBUG3, "*** WARNING *** response type already chosen\n");
```

Comment:
things that say warnings should maybe be debug1, that seems somewhat important

Reply:
Resolved